.github/workflows/scripts/collectPromptRes.js

```diff
@@ -9,22 +9,32 @@ const directories = fs.readdirSync(evalPromptDir, { withFileTypes: true })
     .map(dirent => dirent.name);
 console.log(directories);
 for (const dir of directories) {
-    const outputFile = path.join(path.join(evalPromptDir, dir), 'output.json');
+    const outputFile = path.join(evalPromptDir, dir, 'output.json');
+    const baselineFile = path.join(evalPromptDir, dir, 'baseline.json');
     if (outputFile == null || !fs.existsSync(outputFile))
         continue;
     console.log(outputFile);
     const output = JSON.parse(fs.readFileSync(outputFile, 'utf8'));
-    let body = `⚠️ LLM prompt result for ${dir}
-
-| Success | Failure |
-|---------|---------|
-| ${output.results.stats.successes}      | ${output.results.stats.failures}       |
+    let body = ` LLM prompt result for ${dir}
 
+| From | Success | Failure |  Score  |
+|---------|---------|---------|---------|
+| Output | ${output.results.stats.successes} | ${output.results.stats.failures} | ${output.results.prompts[0].metrics.score} |
```

Comment:
How is `output.results.prompts[0].metrics.score` calculated? It is calculated by promptFoo?

Reply:
`output.results.prompts[0].metrics.score` is calced by promptfoo, Promptfoo aggregates the scores of all the results to obtain this value. Since Promptfoo runs concurrently, it cannot determine the value of each individual test case. Therefore, the total score is used for reference.